kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/MetricsConstants.scala

```diff
@@ -37,6 +37,8 @@ object MetricsConstants {
   final private val THRIFT_BINARY_CONN = KYUUBI + "thrift.binary.connection."
   final private val REST_CONN = KYUUBI + "rest.connection."
 
+  final val SSL_CERT_EXPIRATION = KYUUBI + "ssl.cert.expiration"
```

Comment:
will ssl cert be used in other places? do we need an additional namespace?

Reply:
https://github.com/apache/kyuubi/blob/fd539130764b466778e88144390668a59154b0eb/kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala#L768-L774

Reply:
Currently, the keystore is used for thrift binary and http SSL.

Reply:
Should we add `thrift` ?
```
kyuubi.thrift.ssl.cert.expiration
```

Reply:
Currently, the keystore is not used for RESTful yet.

We use the certificate of Loadbalancer for http SSL instead of maintain the keystore for every kyuubi instance.

Reply:
> Should we add `thrift` ?
> 
> ```
> kyuubi.thrift.ssl.cert.expiration
> ```

sgtm